library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -330,6 +331,19 @@ libspdm_return_t libspdm_handle_large_request(
         return LIBSPDM_STATUS_ERROR_PEER;
     }
 
+    /* Fail if exceed max chunks */
+    min_data_transfer_size = LIBSPDM_MIN(
+        spdm_context->connection_info.capability.data_transfer_size,
+        spdm_context->local_context.capability.sender_data_transfer_size);
+
+    max_chunk_data_transfer_size =
```

Comment:
we should also consult max_spdm_msg_size.

Reply:
The Responder's `max_spdm_msg_size` is already checked in `libspdm_send_spdm_request`. Checking the Requester's `max_spdm_msg_size` doesn't make sense, as the message already exists at this point and so must be small enough to fit.

Reply:
Ok. If it is already checked, just add a comment to say : it is already checked.